bundles/org.openhab.ui/web/src/components/rule/trigger-module-wizard.vue

```diff
@@ -282,6 +283,10 @@ export default {
         case 'timeOfDay':
           this.$emit('typeSelect', 'timer.TimeOfDayTrigger', true)
           break
+        case 'dateTime':
+          this.$emit('typeSelect', 'timer.DateTimeTrigger', true)
+          if (this.currentItem) this.$set(this.currentModule, 'configuration', Object.assign({}, { itemName: this.currentItem.name }))
```

Comment:
`this.currentItem` isn't relevant when you choose a "Time event" trigger, only when you choose "Item event" (then the UI lets you pick an item (put into `this.currentItem` and only then you may choose the trigger type and configuration).

Since the generic config sheet is already displayed for these "Time event" triggers (so you'll be able to choose the item) I suppose there's nothing more to do than remove this line.